docs/fast_data/tutorials/low-code.mdx

```diff
@@ -780,11 +1058,38 @@ Now we have the Projections, the Single View and each configuration ready to be
 
 Now we can [commit](/console/handbooks/project-usage.md#save-changes) our changes, [deploy](/console/handbooks/project-usage.md#deploy-changes), and try our [Fast Data](/fast_data/architecture.md) sending data from the Kafka Topic, and see the result in the Single View using the endpoints.
```

Comment:
at deploy, SVC does not work properly. From logs: "must have required property 'SINGLE_VIEWS_ERRORS_COLLECTION'".
I think that single views errors collection has not been mentioned in tutorial

Reply:
added